packages/aragon-api-react/src/index.js

```diff
@@ -0,0 +1,150 @@
+import {
+  createContext,
+  createElement,
+  useContext,
+  useEffect,
+  useState,
+} from 'react'
+import Aragon, { providers } from '@aragon/api'
+import { map } from 'rxjs/operators'
+
+const postMessage = (name, value) => {
+  window.parent.postMessage({ from: 'app', name, value }, '*')
+}
+
+const requestMenu = () => {
+  postMessage('requestMenu', true)
+}
+
+const AragonApiContext = createContext()
+
+function AragonApi({
+  children,
+  reducer = state => (state === null ? {} : state),
+  onMessage = () => {},
+}) {
+  const [api, setApi] = useState(null)
+  const [connectedAccount, setConnectedAccount] = useState('')
+  const [network, setNetwork] = useState('')
```

Comment:
Should this also start off as `null` or an empty object?

Reply:
Yes, good catch!